main.tf

```diff
@@ -125,6 +94,10 @@ resource "aws_elasticsearch_domain" "default" {
     dedicated_master_count   = "${var.dedicated_master_count}"
     dedicated_master_type    = "${var.dedicated_master_type}"
     zone_awareness_enabled   = "${var.zone_awareness_enabled}"
+
+    zone_awareness_config {
```

Comment:
I'm specifically interested in getting this part merged. @renegaderyu are you still going to polish this PR?
I was wondering if I should wait for you, or cover this specific part in a separate PR

Reply:
@osterman you might want to do that in a separate PR. I'll probably delete this PR since its in conflict and I wasn't sure if you all wanted to drop the `kibana_hostname` or not.